project/depends.scala

```diff
@@ -0,0 +1,22 @@
+import sbt._
+
+object depends {
+
+  def scalatest(scalaVersion: String) =
+    if (scalaVersion.startsWith("2.12")) "org.scalatest" %% "scalatest" % "3.0.0" % Test
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal "2.12" appears 4 times in the file.](https://www.codacy.com/app/pathikrit/better-files/file/3776472232/issues/source?bid=2860326&fileBranchId=3676352#l6)